api-service/cloudify_api/main.py

```diff
@@ -0,0 +1,17 @@
+from cloudify_api import CloudifyAPI
+from cloudify_api.routers import audit as audit_router
+
+
+TITLE = "Cloudify API"
+VERSION = "6.3.0.dev1"
```

Comment:
Can we pull the version string from pkgutils or something, to avoid having it hardcoded in yet another place?